docs/README.md

```diff
@@ -1,2 +1,204 @@
 # jag-file-submission
+
 Generic File Submission API (to be used by the Family Law Act Application at first)
+
+## Introduction
+
+Ready for integrating your application with bcgov efiling service and allow your users to efile document ?
+
+This guide will provide you with all information you need to get started.
+
+### Request a profile
+
+> Follow the following instructions in [Getting started - Onboarding](onboarding.md)
+
+### Start Integrating
+
+Get an oauth token from our Keycloak Server using the following curl
+
+```bash
+curl --location --request POST 'url' \
+--header 'Content-Type: application/x-www-form-urlencoded' \
+--data-urlencode 'client_id=[your client id]' \
+--data-urlencode 'grant_type=client_credentials' \
+--data-urlencode 'client_secret=[your client secret]'
+```
+
+> the secret should be kept safe and not exposed to the public
+
+Start by uploading document(s) to the **efiling hub** api
+
+```bash
+curl --location --request POST 'http://fla-nginx-proxy-qzaydf-dev.pathfinder.gov.bc.ca/api/submission/documents' \
```

Comment:
is our reverse proxy URL meant to be public?

Reply:
removed